ajax/libs/jquery.smartmenus/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "jquery.smartmenus",
+  "filename": "jquery.smartmenus.min.js",
+  "version": "1.0.0",
+  "homepage": "http://www.smartmenus.org/",
+  "authors": "Vasil Dinkov <vasko.dinkov@gmail.com>",
+  "description": "Advanced jQuery website menu plugin. Mobile first, responsive and accessible list-based website menus that work on all devices.",
+  "keywords": [
+    "menu",
+    "navigation",
+    "accessible",
+    "responsive",
+    "mobile",
+    "ui"
+  ],
+  "license": "MIT",
+  "repositories": [
+    {
+      "type": "git",
+      "url": "https://github.com/vadikom/smartmenus"
+    }
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/vadikom/smartmenus.git",
+    "basePath": "dist/",
+    "files": [
+      "addons/**/*",
+      "css/**/*.css",
+      "jquery.smartmenus*.js"
+    ]
```

Comment:
Wow! Thank you for point out this.
This expression is more clear and intelligent!
I will take that. :-D